.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: c
 os: linux
 dist: trusty
+before_install:
+  - sudo apt-get -y install libegl1-mesa-dev libgles2-mesa-dev libsdl2-dev
```

Comment:
try adding libsdl2-2.0 (that's what https://wiki.libsdl.org/Installation tells)

Reply:
Will be trying that now

Reply:
Still failing :(